subworkflows/local/pairalign_m2m/main.nf

```diff
@@ -109,6 +109,8 @@ workflow PAIRALIGN_M2M {
 
     emit:
 
+    train = LAST_TRAIN.out.param_file
```

Comment:
If the `mqc.tsv` files are produced by the `last/train` module, we do not need to export the parameter files.

Reply:
yes